.github/.codecov.yml

```diff
@@ -12,6 +12,12 @@ coverage:
         target: auto
         threshold: 1%
         base: auto
+    patch:
+      default:
+        # basic settings
+        target: auto
+        threshold: 15%
+        base: auto
```

Comment:
the threshold is too large and it may cause problems, could you temporarily ignore the magic number in the file `mnemonic.ts`

```
// TODO comment for these magic numbers
/* eslint-disable @typescript-eslint/no-magic-numbers */
```

Reply:
OK. Set to 1% now.